Source/.clang-tidy

```diff
@@ -54,14 +54,16 @@ CheckOptions:
   - { key: readability-identifier-naming.ClassCase,               value: CamelCase  }
   - { key: readability-identifier-naming.StructCase,              value: CamelCase  }

```

Comment:
1. The Google Style Guide comment no longer applies.
2. `HeaderFilterRegex` above needs to be adjusted (no more SXT directories)

Reply:
SXT?

Reply:
https://github.com/diasurgical/devilutionX/compare/d68a9735555e942e271285cdfc33448964485639..c6ce60ead4df5f16b24b190117aa7a502aafc470

Reply:
Perhaps `(Source|test)`?

Reply:
Sure
https://github.com/diasurgical/devilutionX/compare/c6ce60ead4df5f16b24b190117aa7a502aafc470..41065095ff753d4d7cd73ea860a0c4d218b0fb57